code/ai/aicode.cpp

```diff
@@ -12290,9 +12294,12 @@ void ai_maybe_evade_locked_missile(object *objp, ai_info *aip)
 				case AIM_CHASE:
 					//	Don't always go into evade weapon mode.  Usually, a countermeasure gets launched.
 					// If low on countermeasures, more likely to try to evade.  If 8+, never evade due to low cmeasures.
+                    // tcrayford: add smart secondary evade - when enabled, the AI will use the evasion value as a chance
+					// to evade secondaries instead of never doing it with high countermeasures.
 					if (((((Missiontime >> 18) ^ OBJ_INDEX(objp)) & 3) == 0) || 
 						(objp->phys_info.speed < 40.0f) ||
-						(frand() < 1.0f - (float) shipp->cmeasure_count/8.0f)) {
+						(frand() > (1.0f - aip->ai_smart_secondary_evade_chance)) ||
+						(frand() < (1.0f - (float) (shipp->cmeasure_count/8.0f)))) {
```

Comment:
Is it intended that a ship with both "$Smart Secondary Evade Chance:" *and* fewer than 8 countermeasures gets two chances to decide to evade?

Reply:
ah, good catch. I'll fix it.